.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -2,14 +2,14 @@
 # https://github.com/firebase/firebase-tools
 
 name: Deploy to Firebase Hosting on PR
-'on': pull_request
+on: pull_request
 jobs:
   build_and_preview:
     if: '${{ github.event.pull_request.head.repo.full_name == github.repository }}'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: cd www && yarn install --frozen-lockfile && npm run build
+      - run: cd www && yarn install --immutable && CI=false yarn build
```

Comment:
Why edit the CI?

Reply:
@dherault From Yarn 2 `yarn install --help`:

```
If the `--immutable` option is set (defaults to true on CI), Yarn will abort 
with an error exit code if the lockfile was to be modified (other paths can be 
added using the `immutablePatterns` configuration setting). For backward 
compatibility we offer an alias under the name of `--frozen-lockfile`, but it 
will be removed in a later release.
```

Reply:
We could remove it there as it will be used by default in CI though.

Reply:
`CI=false` is there since create react app treats all warnings as errors in the CI and fails the build.
https://github.com/facebook/create-react-app/blob/f34d88e30c7d8be7181f728d1abc4fd8d5cd07d3/packages/react-error-overlay/build.js#L43-L51

Build output
```
Creating an optimized production build...
Compiled with warnings.

/static/js/main.eb3869c0.js is 7.97 MB, and won't be precached. Configure maximumFileSizeToCacheInBytes to change this limit.

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

File sizes after gzip:

  2.07 MB   build/static/js/main.eb3869c0.js
  36.18 kB  build/static/css/main.b91960b0.css

The bundle size is significantly larger than recommended.
Consider reducing it with code splitting: https://goo.gl/9VhYWB
You can also analyze the project dependencies: https://goo.gl/LeUzfb

The project was built assuming it is hosted at /.
You can control this with the homepage field in your package.json.

The build folder is ready to be deployed.
You may serve it with a static server:

  yarn global add serve
  serve -s build

Find out more about deployment here:

  https://cra.link/deployment
```

One way to solve that would be to introduce lazy loaded views and split the output bundle into separate chunks per view, but this is way more work, hence the `CI=false` solution.